### ISSUE TITLE
Won't work at all

### ISSUE THREAD

**Issue by DibDibs** [yariplus/nodebb-plugin-minecraft-integration#64]
When I click the 'Add a new server' button, nothing happens, it doesn't give me a GUI or anything!

**Comment by yariplus** [comment]
Can you open the browser console and let me know what the error is?

**Comment by DibDibs** [comment]
Hi, there is a screenshot of all the JavaScript errors I encounter attached to this...

![mc int err](https://cloud.githubusercontent.com/assets/8533154/11372480/b97db452-92c5-11e5-9d7b-16277439bc5a.PNG)

**Comment by DibDibs** [comment]
(Sorry for the delay in replying, I didn't expect a response _that_ fast!)

**Comment by yariplus** [comment]
Are you running a Bukkit server?

I'm pushing an update soon which should fix this, but the plugin now requires it's companion Bukkit plugin, bukkit-plugin-nodebb-integration.

If you're not running a Bukkit server, I will make a non-Bukkit version.

**Comment by DibDibs** [comment]
Hi again, I've got the plugin on my bukkit server, but that's not the problem, the problem is, when I click 'add server' on the nodebb admin panel, nothing happens, I just get them JavaScript errors in the console. Do you know what's happening there / is there another method of adding my server to the forum?

Thanks again, DibDibs :)

**Comment by yariplus** [comment]
Thanks,

I'm working on a fix. I just needed to make sure you had the plugin installed.

**Comment by DibDibs** [comment]
Yeah, I do. Thanks for being so helpful :)

**Comment by yariplus** [comment]
fyi, I'm going to re-work that whole section of the admin page. It's really derpy and prone to breaking. (I just did a fake update for testing, and it completely wiped all of the servers I had entered.)

I should be done before Sunday if all goes well.

**Comment by DibDibs** [comment]
Cool, thanks for being so helpful, you're a good dev. Some just ignore you and leave you to it...

**Comment by yariplus** [comment]
Try out the new version 0.6.0 when you get a chance.

**Comment by DibDibs** [comment]
OK, will do
My NodeBb isn't working. I'll let u know when it's fixed :-)

**Comment by yariplus** [comment]
This is pretty old. Create a new issue if this is still a problem.
